README.md

```diff
@@ -9,3 +10,25 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Proof
+
+Suppose graphs $A$ and $B$ are composed of just one node each. 
+
+A = [$n_{1,1}$]
+
+B = [$n_{2,1}$]
+
+These graphs are already isomorphic because $n_{1,1} \rightarrow n_{2,1}$, that's the only mapping that exists here. 
+
+Let's add a new node to each graph and connect it to the other node in that graph, so that the graphs remain completely connected. 
+
+A = [$n_{1,1}$] - $E_{1,1}$ - [$n_{1,2}$]
+
+B = [$n_{2,1}$] - $E_{2,1}$ - [$n_{2,2}$]
+
+These graphs with edges are also isomporphic becasue they satisfy the definition. $f: n_1 \rightarrow n_2$ such that $(n_{1,1}, n_{1,2}) \in E_{1,1}$ iff $(f(n_{1,1}), f(n_{1,2})) \in E_{2,1}$
```

Comment:
But how do you get this mapping $f$?

Reply:
My mapping $f$ gives back the $n_{2,k}$ for every $n_{1,k}$.